utils/release-candidate-testing/python/rc_test.py

```diff
@@ -1,82 +1,15 @@
 # Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
 
 import asyncio
-import os
-import subprocess
-import sys
-from typing import List, Tuple
-
-SCRIPT_FILE = os.path.abspath(f"{__file__}/../../../cluster_manager.py")
+from typing import List
```

Comment:
I think filename should be changed to async_rc_test.py
and make sure you change it also in pypi-cd.yml that calls it